plugins/modules/win_firewall.ps1

```diff
@@ -50,7 +55,20 @@ Try {
                 $result.changed = $true
                 $result.$profile.enabled = $true
             }
-
+            if($null -ne $inbound_action) {
+                $inbound_action = (Get-Culture).TextInfo.ToTitleCase(($inbound_action.ToLower() -replace "_", " ")) -replace " ", ""
+                if ($inbound_action -ne $current_inboundaction) {
+                  Set-NetFirewallProfile -name $profile -DefaultInboundAction $inbound_action -WhatIf:$check_mode
+                  $result.changed = $true
+                }
+            }
+            if($null -ne $outbound_action) {
+                $outbound_action = (Get-Culture).TextInfo.ToTitleCase(($outbound_action.ToLower() -replace "_", " ")) -replace " ", ""
+                if ($null -ne $outbound_action -and $outbound_action -ne $current_outboundaction) {
```

Comment:
No need for the `$null -ne $outbound_action` check as you've already got this in the higher if statement

```suggestion
                if ($outbound_action -ne $current_outboundaction) {
```